src/plugin.js

```diff
@@ -22,7 +23,8 @@ function setPluginOptions (pluginOptions) {
     bootstrapFilename: bootstrapFilename || 'inline.chunk.js',
     templateFilename: templateFilename || 'index.html',
     templatePath: templatePath || 'templates/[name]',
-    htmlTemplatePath: htmlTemplatePath || undefined
+    htmlTemplatePath: htmlTemplatePath || undefined,
+    htmlWebpackPluginOptions: htmlWebpackPluginOptions || {}
```

Comment:
I'd like to have some tests specific for a few of the features used here. I know there are a lot so just a few would be good. 

Also could I convince you to submit an issue (as a reminder to us) to add better tests for html-webpack-plugin options overall.